src/data/boundary.json

```diff
@@ -77,5 +77,18 @@
 			}
 		]
 	},
-	"navigationHeaderItems": {}
+	"navigationHeaderItems": {
+		"documentation": [
+			{
+				"label": "General Documentation",
+				"icon": "docs",
+				"pathSuffix": "docs"
+			},
+			{
+				"label": "API",
+				"icon": "api",
+				"pathSuffix": "api-docs"
+			}
```

Comment:
Can we go ahead and add this to `rootDocsPaths` so it's ready?

Reply:
✅ Added!